prometheus-exporters/aci-suite/values.yaml

```diff
@@ -0,0 +1,33 @@
+# Global settings shared by all subcharts
+global:
```

Comment:
an option which needs to be set here is linkerd:
https://github.com/sapcc/helm-charts/blob/master/system/vmware-monitoring/values.yaml#L12

Also the image repository stuff partially needs to go in here (like domain). See my comment above.